app/helpers/document_helper.rb

```diff
@@ -217,8 +217,11 @@ def part_of_metadata(document, policies = [], sector_tag_finder = nil)
       part_of += array_of_links_to_topical_events(document.topical_events)
     end
 
-    if sector_tag_finder && (all_sectors = sector_tag_finder.sectors_and_subsectors).any?
-      part_of += array_of_links_to_sectors(all_sectors)
+    if sector_tag_finder && (tagged_topics = sector_tag_finder.topics).any?
```

Comment:
Nitpick: I know this was there before, but as a general style thing I'm not a fan of assignment in the middle of expressions. It's a really common mistake to type `=` when you mean `==`, so whenever I read this I do a double take and have to check whether its actually intentional.

Python [explicitly made this a syntax error](https://docs.python.org/2/faq/design.html#why-can-t-i-use-an-assignment-in-an-expression), and having just spent ~30 minutes debugging a similar problem, I really miss this feature in ruby!